packages/perspective-viewer-datagrid/src/js/event_handlers/keydown/edit_keydown.js

```diff
@@ -82,6 +84,17 @@ const moveSelection = lock(async function (
     }
 });
 
+function isLastCell(model, table, target) {
+    const meta = table.getMeta(target);
+    return meta.y === model._num_rows - 1 && meta.x === model._column_paths.length - 1;
+}
+
+function saveCellContent(model, table, target) {
```

Comment:
This is not needed, the cell contents are already saved on blur - and this can be an expensive call if the `View` is remote.